ixml.ixml

```diff
@@ -23,18 +25,19 @@
       repeat1: factor, "+", S, sep?.
        option: factor, "?", S.
           sep: factor.
-  nonterminal: (mark, S)?, name, S.
+  nonterminal: (mOrP, S)?, name, S.
```

Comment:
It might fit more easily into the style of the existing grammar is instead of `mOrP` we named the nonterminal `annotation` or `annot` (and similarly `tannot[ation]` for terminals).

Alternatively, replace `(mark, S)?` and `(tmark, S)?` in the current grammar with `marks` and `tmarks`, and define them as `(mark, S)?, (pragma, S)?` and `(tmark, S)?, (pragma, S)?`, respectively.

Reply:
Definitely prefer annotation and tannotation than mOrP: I had hoped someone would come up with a better name!